pages/GLAM3D/identify.md

```diff
@@ -498,11 +498,11 @@ We do not currently have a good example of an Open Access representation of a pe
 
 ## 5.2. Cultural and Other Non-Legal Considerations
 
-There may also be non-legal factors to consider when selecting works to digitize and make available to the public.[^44] While these factors may vary widely depending on the nature of your collection, one factor to consider is how you can effectively propagate an awareness of the cultural context that created a work in the first place.
+There may also be non-legal factors to consider when selecting cultural resources to digitize and make available to the public.[^44] These factors may vary widely depending on the nature of your collection. One factor to consider is how you can effectively propagate an awareness of the cultural resource's original cultural context.
 
-The best way to address these concerns will vary widely, depending on the nature of the cultural resources and the sensitivities involved. In some cases, it may be beneficial to consult with the cultural originators (or their descendants) of a work to identify concerns that the communities may have in relation to digitization and Open Access programs.
+The best way to address these concerns will vary widely, depending on the nature of the cultural resources and the sensitivities involved. In some cases, it may be beneficial to consult with the cultural originators (or their descendants) of a cultural resource to identify concerns that the communities may have in relation to digitization and Open Access programs.
 
-One innovative approach to cultural resources originating with indigenous communities has been the [Traditional Knowledge (TK) labelling system](http://localcontexts.org/tk-labels/).[^55] The TK label system makes it easy for indigenous communities to communicate context and preferred uses to others. These labels are not intended to be legal restrictions on users. Instead, they are designed to communicate additional cultural context to users interested in incorporating that context into their own exploration of the resource. TK labels can easily be incorporated into the presentation of the digitized version of the cultural asset.[^45]
+One innovative approach to cultural resources originating with indigenous communities has been the [Traditional Knowledge (TK) labelling system](http://localcontexts.org/tk-labels/).[^55] The TK label system makes it easy for indigenous communities to communicate context and preferred uses to others. These labels are not intended to be legal restrictions on users. Instead, they are designed to communicate additional cultural context to users interested in incorporating that context into their own exploration of the resource. TK labels can easily be incorporated into the presentation of the digitized version of the cultural asset.[^45] Culutral Institution Notices are tools specifically designed to be used by cultural institutions as they engage in collaborative partnerships with communities to address problematic histories, unclear provenance, and incomplete attribution.
```

Comment:
Cultural instead of Culutural. Could be a typo from the spreadsheet.

Reply:
Thanks for catching it. I just updated it in a new pull requests. I've never actually done that, so we'll see if it works.

Reply:
Looks like it did so I'll push the changes. Technology!